egs/iwslt22_ta/ST/pruned_transducer_stateless5/train.py

```
@@ -0,0 +1,1290 @@
+            # clip_grad_norm_(model.parameters(), 5.0, 2.0)
+            optimizer.step()
+            optimizer.zero_grad()
+        except Exception as e:
+            if "CUDA out of memory" in str(e):
+                logging.error(
+                    "Your GPU ran out of memory with the current "
+                    "max_duration setting. We recommend decreasing "
+                    "max_duration and trying again.\n"
+                    f"Failing criterion: {criterion} "
+                    f"(={crit_values[criterion]}) ..."
+                )
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)